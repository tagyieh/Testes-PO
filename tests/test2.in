4
3
969001
9
969014
VOICE
11
10
20
7
0
0
5
5
4
0
0

