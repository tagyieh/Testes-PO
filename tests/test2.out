MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: VOICE|1|969001|969014|0|0|ONGOING
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
10 - Terminar comunicação activa
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 400 unidades.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Valores para o terminal '969001':0 (pagamentos),  400 (dívidas).
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|400
CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
CONSULTAS
1 - Mostrar todas as comunicações
2 - Mostrar comunicações feitas por um cliente
3 - Mostrar comunicações recebidas por um cliente
4 - Mostrar clientes sem dívidas
5 - Mostrar clientes com dívidas
6 - Mostrar terminais sem actividade
7 - Mostrar terminais com saldo positivo
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 